test/jdk/javax/management/remote/mandatory/connection/DefaultAgentFilterTest.java

```diff
@@ -282,6 +291,28 @@ public static void main(String[] args) throws Exception {
             System.out.println("----\tTest FAILED !!");
             throw ex;
         }
+        try {
+            // Use default filter, should fail with: java.io.InvalidClassException: filter status: REJECTED
+            testDefaultAgent(null /* no properties file */);
+            throw new RuntimeException("---" + DefaultAgentFilterTest.class.getName() + " - No exception reported");
```

Comment:
Not that it matters much but traditionally we would throw AssertionError in such cases...